scripts/add_brownfield.py

```diff
@@ -0,0 +1,183 @@
+# coding: utf-8
+
+import logging
+logger = logging.getLogger(__name__)
+import pandas as pd
+idx = pd.IndexSlice
+
+import numpy as np
+import scipy as sp
+import xarray as xr
+import re, os
+
+from six import iteritems, string_types
+
+import pypsa
+
+import yaml
+
+import pytz
+
+from vresutils.costdata import annuity
+
+from add_existing_baseyear import add_power_capacities_installed_before_baseyear 
+
+from add_existing_baseyear import add_heating_capacities_installed_before_baseyear 
+
+from add_existing_baseyear import prepare_costs
+
+#First tell PyPSA that links can have multiple outputs by
+#overriding the component_attrs. This can be done for
+#as many buses as you need with format busi for i = 2,3,4,5,....
+#See https://pypsa.org/doc/components.html#link-with-multiple-outputs-or-inputs
+
+override_component_attrs = pypsa.descriptors.Dict({k : v.copy() for k,v in pypsa.components.component_attrs.items()})
+override_component_attrs["Link"].loc["bus2"] = ["string",np.nan,np.nan,"2nd bus","Input (optional)"]
+override_component_attrs["Link"].loc["bus3"] = ["string",np.nan,np.nan,"3rd bus","Input (optional)"]
+override_component_attrs["Link"].loc["efficiency2"] = ["static or series","per unit",1.,"2nd bus efficiency","Input (optional)"]
+override_component_attrs["Link"].loc["efficiency3"] = ["static or series","per unit",1.,"3rd bus efficiency","Input (optional)"]
+override_component_attrs["Link"].loc["p2"] = ["series","MW",0.,"2nd bus output","Output"]
+override_component_attrs["Link"].loc["p3"] = ["series","MW",0.,"3rd bus output","Output"]
+
+
+def prepare_costs():
```

Comment:
Do we need this function prepare_costs() defined here if we already imported it from add_existing_baseyear?

Reply:
We can import it from add_existing_baseyear. I have a meeting now but I will include it tomorrow

Reply:
This is done in [this commit](https://github.com/PyPSA/pypsa-eur-sec/pull/63/commits/7d1500604fd1027a7d8952daa4cc40c58444ff66). 

prepare_costs() function has been modified to include the 2 levels multindex files produced in the t[echnology_data repository](https://github.com/lisazeyen/technology_data/tree/master/outputs).

hydrogen underground storage is replaced by gas storage until the name is updated in the technolgy_data repository